lib/src/ui/pages/authentication/field_error_message_provider.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/domain/authorization/authentication_field_error.dart';
+import 'package:allthenews/src/ui/common/message_provider.dart';
+
+class FieldErrorMessageProvider extends MessageProvider {
```

Comment:
`AuthFieldErrorMessageProvider`

Reply:
nie jest powiedziane, że tylko błędy autentykacji tu będą, obecnie tak jest, ale jak dojdzie walidacja innych pól to tu też będzie można je określić. Myślę, że nie warto tego rozdzielać, bo te błędy mogą się powielać (np pustość). Taki był zamysł i zmieniłbym nazwę `AuthenticationFieldError` na `FieldError`